actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -963,12 +963,12 @@ def deep_dup
 
     # Returns parameter value for the given +key+ separated by +delimiter+.
     #
-    #   params = ActionController::Parameters.new(id: "1_123", tags: "ruby,rails")
+    #   params = ActionController::Parameters.new(id: "1_123", tags: "ruby,rails,")
```

Comment:
I don't mind adding something that mention this edge case, but I don't think it should replace the existing example, as the first example in a doc should pretty much always be the happy path.

Reply:
Agreed. Extracted as a separate note to the method's doc.